src/components/change-requests/new-change-request/new-change-request-page/standard-form-fields/standard-form-fields.tsx

```diff
@@ -7,6 +7,7 @@ import { Row, Col, Form, InputGroup, FormControl } from 'react-bootstrap';
 import { exampleAllWorkPackages } from '../../../../../test-support/test-data/work-packages.stub';
 import { wbsPipe } from '../../../../../shared/pipes';
 import styles from './standard-form-fields.module.css';
+import { ChangeRequestReason } from 'utils/lib/types/change-request-types';
```

Comment:
```suggestion
import { ChangeRequestReason } from 'utils';
```

things from utils should only ever be imported with `'utils'`